.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+repos:
+-   repo: https://github.com/ambv/black
+    rev: 18.9b0
```

Comment:
These versions should be updated and the tools rerun before merge to keep out as much churn as we can.

Reply:
`pre-commit autoupdate` will set the latest versions, then `pre-commit run --all-files` will reapply them.